resource-cleanup.md

```diff
@@ -0,0 +1,32 @@
+# Cleaning up old resources
+
+When an app is pushed using Application Service Adapter, a number of resources are created that can build up over time.
+Each source code upload results in a CFPackage resource on the cluster, and a corresponding container image holding the source in the sources repository.
+Each build of the sources results in a CFBuild resource, and a corresponding container image holding the built image in the droplets repository.
+To minimize container registry storage costs, Application Service Adapter will delete older CFPackage and CFBuild resources, and ensures the container registry images are removed when these resources are removed.
```

Comment:
@kieron-dev do you think "ensures the container registry images are removed" is too strong of language here?

My understanding is that the registry image deletion is "best-effort" so there is a chance that the deletion will fail and the container images will remain. I think this is the right tradeoff for better availability of TAS Adapter, but we should probably make that expectation explicit.

Reply:
Thanks @tcdowney. I've altered that line and added another clarifying sentence.

Reply:
We'll leave merging to you

Reply:
@tcdowney Please let me know if it looks good to you, and I'll merge and get to editing it. Thanks.